products/ruby.md

```diff
@@ -24,8 +24,8 @@ identifiers:
 releases:
 -   releaseCycle: "3.2"
     eol: 2026-03-31
-    latest: "3.2.0"
-    latestReleaseDate: 2022-12-25
+    latest: "3.2.1"
```

Comment:
Changelog not yet published on https://rubychangelog.com. But the release [has been announced](https://www.ruby-lang.org/en/news/2023/02/08/ruby-3-2-1-released/).